js/worker.js

```
@@ -5034,10 +5033,11 @@ async function convertAndOrganiseFiles(threadLimit) {
   const fileProgressMap = {};
   const conversions = []; // Array to hold the conversion promises
 
-  // Query the files table to get the necessary data
-  const rows = await db.allAsync(
-    "SELECT f.id, f.name, f.duration, f.filestart, l.place FROM files f LEFT JOIN locations l ON f.locationID = l.id"
-  );
+  // Query the files & records table to get the necessary data
+  const query = "SELECT f.id, f.name, f.duration, f.filestart, l.place FROM files f LEFT JOIN locations l ON f.locationID = l.id";
+  // If jsut saving files with records
+  if (STATE.libraryClips) query += " WHERE EXISTS (SELECT 1 FROM records r WHERE r.fileID = f.id)"
+  const rows = await db.allAsync(query);
```

Biome (1.9.4)

[error] Can't assign query because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)